.eslintrc

```diff
@@ -1,3 +1,5 @@
+extends: eslint:recommended
```

Comment:
That's true, assuming we notice of course. Either way works for me. I doesn't sound likely that `eslint:recommended` will change anyway.